<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>server</class>
 <widget class="QDialog" name="server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QPushButton" name="CloseServer">
     <property name="text">
      <string>CloseServer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="MsgForServer"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="OpenServer">
     <property name="text">
      <string>OpenServer</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="ChangeSettings">
     <property name="text">
      <string>ChangeSettings</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="Exit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
